app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, origin_file, path = command.split()
```

Comment:
Validate command length before unpacking

Reply:
not fixed